src/bindings/jna/README.md

```diff
@@ -31,34 +35,18 @@ which contains the libelektra4j subdirectory that corresponds to the
 libelektra.jar), e.g.:
 
 ```sh
-export CLASSPATH="/usr/share/java/libelektra-*version*.jar:/usr/share/java/jna.jar"
-export CLASSPATH="~e/src/bindings/jna:/usr/share/java/jna.jar"
+export CLASSPATH=".:/usr/share/java/libelektra4j.jar:/usr/share/java/jna.jar"
 ```
 
-to set it permanently for your user, you can use:
+Or, if you want to use Elektra from cmake's build directory, use
+`
```

Comment:
Did something go wrong here or is this intentional?

Reply:
Yes, this is an error, initially I wanted to recommend to use `scripts/dev/run_env` but it does not work so nice (installing with mvn seems to be the better option).